src/frontend/src/routes/(new-styling)/manage/(authenticated)/+layout.svelte

```diff
@@ -0,0 +1,245 @@
+    identityInfo.reset();
+    toaster.success({
+      title: "You're all set. Your identity has been created.",
+      duration: 2000,
+    });
+    lastUsedIdentitiesStore.selectIdentity(identityNumber);
+    identityInfo.fetch();
+    await gotoManage();
+    isAuthDialogOpen = false;
+  };
+
+  const handleSwitchIdentity = async (identityNumber: bigint) => {
+    const authLastUsedFlow = new AuthLastUsedFlow();
+    const chosenIdentity =
+      $lastUsedIdentitiesStore.identities[Number(identityNumber)];
+    await authLastUsedFlow.authenticate(chosenIdentity).catch((e) => {
+      throw new Error("Could not authenticate");
+    });
+    identityInfo.reset();
+    lastUsedIdentitiesStore.selectIdentity(identityNumber);
+    identityInfo.fetch();
+    await gotoManage();
+    isIdentityPopoverOpen = false;
+  };
+
+  beforeNavigate((nav) => {
+    const fromPathName = nav.from?.url.pathname;
+    const toPathName = nav.to?.url.pathname;
+
+    let anchors;
+    if (isDesktopViewport()) {
+      if (!sideBarGroupRef) return;
+      anchors = Array.from(sideBarGroupRef.querySelectorAll("a"));
+    } else {
+      if (!tabsGroupRef) return;
+      anchors = Array.from(tabsGroupRef.querySelectorAll("a"));
+    }
+
+    const fromElement = anchors.find(
+      (a) => a.getAttribute("href") === fromPathName,
+    );
+    if (!fromElement) return;
+    const fromPosition = anchors.indexOf(fromElement);
+
+    const toElement = anchors.find(
+      (a) => a.getAttribute("href") === toPathName,
+    );
+    if (!toElement) return;
+    const toPosition = anchors.indexOf(toElement);
+
+    if (fromPosition > toPosition) {
+      if (isDesktopViewport()) {
+        animationDirection = "up";
+      } else {
+        animationDirection = "left";
+      }
+    } else {
+      if (isDesktopViewport()) {
+        animationDirection = "down";
+      } else {
+        animationDirection = "right";
+      }
+    }
+  });
+</script>
+
+<SideBarOrTabs>
+  {#snippet sidebarElements()}
+    <SideBarElementGroup bind:bindableGroupRef={sideBarGroupRef}>
+      <SideBarElement href="/manage">
+        <LucideHome size="1rem" class="stroke-fg-quaternary" />
+        <h1>Home</h1>
+      </SideBarElement>
+      <SideBarElement class="text-text-primary" href="/manage/security">
+        <Shield size="1rem" class="stroke-fg-quaternary" />
+        <h1>Security</h1>
+      </SideBarElement>
+    </SideBarElementGroup>
+  {/snippet}
+
+  {#snippet tabElements()}
+    <TabElementGroup bind:bindableGroupRef={tabsGroupRef}>
+      <TabElement class="text-text-primary" href="/manage">
+        <h1>Home</h1>
+      </TabElement>
+      <TabElement class="text-text-primary" href="/manage/security">
+        <h1>Security</h1>
+      </TabElement>
+    </TabElementGroup>
+  {/snippet}
+
+  {#snippet content()}
+    {#key page.url.pathname}
+      <div
+        bind:this={divRef}
+        in:fly={{
+          y:
+            animationDirection === "up"
```

Comment:
How about creating a helper  or state for the `y` and `x` values?  It would be easier to read the render part this way.